### DISCUSSION TITLE
Feature request - "copy-paste augmentation"

### DISCUSSION THREAD

**Discussion by gofmans** [Project-MONAI/MONAI#4199] [Ideas]
Hi,

I recently encountered an interesting data augmentation technique that could be relevant for segmentation tasks, it's called "copy-paste" augmentation as presented in:  https://arxiv.org/pdf/2012.07177v2.pdf. 

Here is an image from the paper to better illustrate the concept:
![image](https://user-images.githubusercontent.com/26842519/165926531-0b7691ad-29f7-4edc-a936-7aa9a53f3ceb.png)

The paper deals with semantic segmentation tasks in regular 2D images where this kind of augmentation makes more sense (croping and pasting different objects into unrelated scenes) but it could also help in detecting tumors and such in the medical domain.

According to the paper, this could be useful, especially for problems with few data examples as is the case in most medical imaging problems.

Here is another paper that uses a similar technique for lesions segmentation:
https://arxiv.org/ftp/arxiv/papers/2203/2203.10507.pdf

Correct me if I'm wrong but I couldn't find anything like that in the current MONAI augmentations so I would like you guys to consider adding it.

Thanks for the great work,

Sasha

**Comment by wyli** [comment]
thanks, I'm creating a new feature request for this.
